### ISSUE TITLE
Standalone Pyas2 Server

### ISSUE THREAD

**Issue by abhishek-ram** [abhishek-ram/django-pyas2#32]
I have noticed that people are finding it difficult sometimes to setup django-pyas2 because they have to setup django, settings and so on. I was wondering if we should have a standalone version of django-pyas2 where the django project is all setup and you just need to run a bin command like `run_pyas2_server` to get it running.

What do you guys think?

**Comment by chadgates** [comment]
I find that installing correct python versions, virtual environments and correct libraries to have pip install all the dependencies correctly are more difficult than setting up the django and the project in these environments. So not sure if just a stand-alone would actually make it easier for these users having that struggle. Maybe a dockerized stand-alone version would be more suitable? Or even freezing (https://packaging.python.org/overview/#bringing-your-own-python-executable) ?

**Comment by abhishek-ram** [comment]
Yeah I we would need to dockerize it, but to do that we would need to have a django project set up for it right...

**Comment by chadgates** [comment]
I see two options: 
- create a separate repo with a stand-alone project to be run as dockerized version
- or scripting the creation of the project as part of a dockerfile within this repo.

**Comment by abhishek-ram** [comment]
I think we should set up a separate repo for doing this.

**Comment by chadgates** [comment]
I can contribute to that repo. I tend to use cookiecutter-django style of Django projects, so that could be a starting point.
